packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -980,27 +980,26 @@ describe('ListView', function () {
         expect(onAction).not.toHaveBeenCalled();
       });
```

Comment:
why were there test changes necessary?

Reply:
We had a lot of tests that were not true to real life. Firing keyboard events without focus being on the element for example. The ShadowTreeWalker enforces a certain condition in order to work, which is totally fine in the real world, but we could and did easily break in testing. 
https://github.com/adobe/react-spectrum/pull/6046/files#diff-582a0300fa3cc6033754f65cd6965774fe194ab0b85211e4e400eef0656ecc46R78

Reply:
we'll probably want to call that out when we release this so that others who've done the same in their own tests know that they need to update their tests in a similar fashion since it is non-obvious why their tests may suddenly start failing otherwise